lib/graphql/card.ts

```diff
@@ -52,6 +53,16 @@ const GET_CARD = `
   }
 `;
 
+const CREATE_CARD = `mutation createCard(
+  $cardType: CardType!
+) {
+  createCard(
+    cardType: $cardType
```

Comment:
I was under the impression that we changed that in the backend?

Reply:
You are absolutely correct, thank you for spotting it.

I forgot to update the query and it's not the first time 😬 will be more careful about it next time.

I have updated it and double checked that it works against current develop backend now: d7bf808bc9559770462e6470d9703637cc93aa85